docs/guide-disappearance.mdx

```diff
@@ -64,7 +64,7 @@ test('movie title goes away', async () => {
   // note use of queryBy instead of getBy to return null
   // instead of throwing in the query itself
   await waitFor(() => {
-    expect(queryByText('i, robot')).not.toBeInTheDocument()
+    expect(screen.queryByText('i, robot')).not.toBeInTheDocument()
```

Comment:
Same here.

Reply:
findBy would throw an error here because the element is being asserted as _not_ present. The previous query also uses queryBy for symmetry in the example.

Reply:
Oh right, good catch. Thanks for pointing this out 👍